docs/docs/guides/become-an-operator.md

```diff
@@ -0,0 +1,141 @@
+
+Scroll down on the Operator page and find the "Operator's node addresses" section. Click the "Add node address" button, paste in the address of your node wallet, click the button in the dialog and then don't forget to click the Save button.
+On the top section of the Operator page, take note of your "Contract" address. You'll need it soon when configuring your nodes.
+
+With that done, your node is now aware of your Operator, but your Operator is not aware of your node. To fix that, add your node address to your Operator in your Operator page, in the Streamr Hub UI. Your node address is the public 0x ethereum address that matches with your node's signing private key and it's displayed when your node starts. Repeat for as many nodes as you have in your fleet.
+
+### Step 4: Fund your nodes
+You’ll need a bit of `MATIC` (the gas token of Polygon) in your node’s wallets as Streamr nodes that participate in sponsored streams will periodically make transactions. 1 to 5 `MATIC` is recommended since the nodes will be making a few transactions per day. If the node runs out of gas while they’re a part of an active Sponsorship, then a penalty may be applied to your unclaimed earnings. See [Operator value maintenance](../streamr-network/network-roles/operators#operator-maintenance).
+
+### Step 5: Fund your Operator
+At this point the Operator is paired with your funded node(s). Now you'll need to fund your Operator Contract with DATA tokens. There's no minimum amount but if you delegate too little, then your capacity to accept delegations will be limited since the owner of the Operator (you) must maintain at least a 10% owner stake in your Operator.
+
+### Step 6: Join sponsorships
+You’re now in a position to join Sponsorships. Go back to the Hub, go to Network -> Sponsorships and find a Sponsorship you want your Operator to start working on. Click the "Join as Operator" button and select your stake. Note there is a minimum stake of 5000 `DATA` tokens.
+
+## The Mumbai test environment
+You can configure your node to connect with the Mumbai testnet (not to be confused with the Streamr incentivized testnets that will run on Polygon with real tokens). The Mumbai environment is handy for testing Operator..operations with fake tokens.
+
+Mumbai has it's own UI - the [Mumbai Hub](https://mumbai.streamr.network). Your node will need to be configured with the [Mumbai config](#mumbai-testing-environment-node-config)
+
+You'll need Mumbai `MATIC` - widely available with [public faucets](https://mumbaifaucet.com) and you'll need `DATATEST` tokens (the Mumbai network's worthless `DATA` tokens) - you can ask for those on the [Streamr Discord](https://discord.gg/gZAm8P7hK8) and a community admin will send you some to your address. 
```

Comment:
In the UI and in the token itself, the symbol is `TEST` - recommend to use it here too

```suggestion
You'll need Mumbai `MATIC` - widely available with [public faucets](https://mumbaifaucet.com) and you'll need `TEST` tokens (the Mumbai network's worthless `DATA` tokens) - you can ask for those on the [Streamr Discord](https://discord.gg/gZAm8P7hK8) and a community admin will send you some to your address. 
```